crabs/tracker/utils/tracking.py

```diff
@@ -47,6 +47,7 @@ def write_tracked_bbox_to_csv(
     frame: np.ndarray,
     frame_name: str,
     csv_writer: Any,
+    pred_score: np.ndarray,
```

Comment:
Could `pred_score` be just a float?

(As in, do we need it to be an array?)

Reply:
Yes, because we have red_score for every crabs in a frame?